README.md

```diff
@@ -71,7 +73,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+controller method
 ```
```

Comment:
Helper method, because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model mehtods concern how data goes to and comes from the database.